.env.example

```diff
@@ -106,4 +110,4 @@ DELTA_SYNC_INTERVAL_SECONDS=3600
 # LDAP_USER=<LDAP_SERVICE_ACCOUNT_USERNAME>
 # LDAP_PASS=<LDAP_SERVICE_ACCOUNT_PASSWORD>
 # GROUP_BASE_DN=<GROUP_BASE_DN>
-# USER_BASE_DN=<USER_BASE_DN>
+# USER_BASE_DN=<USER_BASE_DN>
```

Comment:
Should add new line at end of file to mark end, I think running the formatter should fix it automatically?